services/server/src/models/game/game.js

```diff
@@ -43,6 +43,14 @@ const GameSchema = new Schema({
       message: '{VALUE} is not a non-negative integer value',
     },
   },
+  customURL: {
+    type: String,
```

Comment:
we should have an index here as well to make it easier to query

Reply:
✍️